sklearn/decomposition/tests/test_pca.py

```diff
@@ -215,16 +216,31 @@ def test_explained_variance():
     # compare to empirical variances
     X_pca = pca.transform(X)
     assert_array_almost_equal(pca.explained_variance_,
-                              np.var(X_pca, axis=0))
+                              np.var(X_pca, ddof=1, axis=0))
 
     X_pca = apca.transform(X)
     assert_array_almost_equal(apca.explained_variance_,
-                              np.var(X_pca, axis=0))
+                              np.var(X_pca, ddof=1, axis=0))
 
     X_rpca = rpca.transform(X)
-    assert_array_almost_equal(rpca.explained_variance_, np.var(X_rpca, axis=0),
+    assert_array_almost_equal(rpca.explained_variance_,
+                              np.var(X_rpca, ddof=1, axis=0),
                               decimal=1)
 
+    # Another way to run this part (according to the original definition)
+    # compare to empirical variances
+    # expected_result = np.linalg.eig(np.cov(X, rowvar=False))[0]
+    # expected_result = sorted(expected_result, reverse=True)[:2]
+    # X_pca = pca.transform(X)
+    # assert_array_almost_equal(pca.explained_variance_, expected_result)
+
+    # X_pca = apca.transform(X)
+    # assert_array_almost_equal(apca.explained_variance_, expected_result)
+
+    # X_rpca = rpca.transform(X)
+    # assert_array_almost_equal(rpca.explained_variance_,
+    #                           expected_result, decimal=1)
```

Comment:
@agramfort Thanks. From my perspective, I think the test is necessary because it is based on the original definition, so I uncomment it, but I would like to know your advise.